examples/rvc-app/README.md

```diff
@@ -0,0 +1,132 @@
+# RVC example app
+
+This example app is meant to demonstrate an implementation of a Matter Robotic
+Vacuum Cleaner device.
+
+## State machine
+
+Below is a diagram describing the state machine for this app.
+
+![state machine](RVC_app_state_diagram.png)
+
+This app can support most of the tests in the test plans.
+
+## Out-of-band messages
+
+Out-of-band messages are available to simulate typical device behaviors and
+allow the app to navigate to all the states. To send an out-of-band message,
+echo the JSON message to the `/tmp/chip_rvc_fifq_<PID>` file. The JSON message
+must have a `"Name"` key that contains the command name. This name is shown in
+the state machine diagram above. Example
+`echo '{"Name": "Charged"}' > /tmp/chip_rvc_fifo_42`.
+
+### ErrorEvent message
+
+The error event message requires the additional key `"Error"` which specifies
+the error state ID. This can be one of UnableToStartOrResume,
+UnableToCompleteOperation, CommandInvalidInState, FailedToFindChargingDock,
+Stuck, DustBinMissing, DustBinFull, WaterTankEmpty, WaterTankMissing,
+WaterTankLidOpen, MopCleaningPadMissing
+
+## Testing
+
+PICS files that detail what this app supports testing are available in the
```

Comment:
I think the idea is for the manufacturers to provide a single file that contains all PICS codes that describe the capabilities of the DUT. One advantage is that you can use the same file name when you run tests, so you don't have to remember the different names of the different tests :)

Reply:
OK. I was not aware of this and since the UI generated them separately I didn't think to combine them. I have combined them in [this commit](https://github.com/project-chip/connectedhomeip/pull/29159/commits/6111b538dd89f0b3b6cfa8e4a812714b02a61b65). Should the XML files also be combined into one?

Reply:
These aren't the Test Plan XML's though, they are the output of PICS Tool to load into the test harness. Perhaps just named badly @hicklin